DSP_Plugins.GalacticScale/Scripts/PatchPlanetSize/PatchNearColliderLogic.cs

```diff
@@ -0,0 +1,127 @@
+﻿using HarmonyLib;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using Patch = GalacticScale.Scripts.PatchPlanetSize.PatchForPlanetSize;
+
+namespace GalacticScale.Scripts.PatchPlanetSize
+{
+    [HarmonyPatch(typeof(NearColliderLogic))]
+    static class PatchNearColliderLogic
+    {
+        public static int offset = -20;
+        public static List<GameObject> markers;
+
+        [HarmonyPostfix]
+        [HarmonyPatch("Update")]
+        public static void Update()
+        {
+            int o = offset;
+            if ((Input.GetKeyUp(KeyCode.Equals) || Input.GetKeyUp(KeyCode.KeypadPlus)) && VFInput.alt)
+                ++offset;
+            if ((Input.GetKeyUp(KeyCode.Minus) || Input.GetKeyUp(KeyCode.KeypadMinus)) && VFInput.alt)
+                --offset;
+            if (o != offset) Patch.Debug("Mining AreaRadius Offset Changed to " + offset, BepInEx.Logging.LogLevel.Message, true);
+
+        }
+        [HarmonyPostfix]
+        [HarmonyPatch("MarkActivePos")]
+        public static void MarkActivePos(Vector3 pos, ref NearColliderLogic __instance, ref int[] ___activeColHashes, ref int ___activeColHashCount)
+        {  
+            if (VFInput.alt && Input.GetKeyUp(KeyCode.Delete))
```

Comment:
why input needed?

Reply:
Debugging mostly, I could comment those out but I would prefer not to delete them entirely- for next time they change the file and we need to tweak things.

Reply:
You could put preprocessor conditions around it. (#if DEBUG and #endif)

That way the code would only be compiled into febug builds.